packages/prettier-config/index.json

```diff
@@ -3,15 +3,17 @@
   "arrowParens": "avoid",
   "bracketSpacing": false,
   "endOfLine": "lf",
+  "htmlWhitespaceSensitivity": "css",
   "jsxBracketSameLine": false,
+  "jsxSingleQuote": false,
   "printWidth": 120,
   "proseWrap": "never",
   "quoteProps": "as-needed",
   "requirePragma": false,
   "semi": true,
   "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": "es5",
+  "trailingComma": "all",
```

Comment:
Can someone remind me why we didn't have `all` here?

Reply:
Because it's annoying to have commas in function arguments :)

Reply:
Define "annoying" :) Can you give me an example on why it's annoying to have the comma there?

Reply:
https://vote4code.com is still up and running.. 🗡